plio/urls.py

```diff
@@ -27,7 +27,7 @@
     verify_otp,
     get_by_access_token,
     retrieve_analytics_app_access_token,
-    convert_third_party_access_token,
+    convert_api_key_to_token,
```

Comment:
should probably change this endpoint name - something like `get_external_auth_access_token`